spec/models/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before :all do
+    @user = User.create(username('test'))
```

Comment:
kindly fix this user creation it's causing errors , `User.create(username: "test")`